### PR TITLE
Fix connection tests (JSTPConnectionTest) style

### PR DESCRIPTION
**Pull request by lundibundi**
* line length
* redundant initializers
* chain calls

### CODE REVIEW

File: jstp/src/test/java/com/metarhia/jstp/connection/JSTPConnectionTest.java [hunk 1]
```diff
@@ -495,5 +501,4 @@ public void invoke(JSValue packet) {
 
     assertTrue(test[0]);
   }
-
 }
```
Comment (aqrln) [inline jstp/src/test/java/com/metarhia/jstp/connection/JSTPConnectionTest.java]: No newline at the end of file.
A minor nit from a random passerby :)

File: jstp/src/test/java/com/metarhia/jstp/connection/JSTPConnectionTest.java [hunk 2]
```diff
@@ -390,13 +394,15 @@ public void sendEscapedCharacters() throws Exception {
...
-            .call("auth", "authorize", new JSArray(new Object[]{"+380962415331", "hellokitty1337"}),
+            .call("auth", "authorize",
+                new JSArray("+380962415331", "hellokitty1337"),
                 new ManualHandler() {
                   @Override
                   public void invoke(JSValue packet) {
                     JSObject userData = new JSObject();
                     userData.put("nickname", "\n\tnyaaaaaa'aaa'[((:’ –( :-)) :-| :~ =:O)],");
-                    connection.call("profile", "update", new JSArray(new Object[]{userData}),
+                    connection.call("profile", "update",
```
Comment (lidaamber) [inline jstp/src/test/java/com/metarhia/jstp/connection/JSTPConnectionTest.java]: To my mind, it looks worse after changes. Maybe, we should less strict about lines length? :)

File: jstp/src/test/java/com/metarhia/jstp/connection/JSTPConnectionTest.java (hunk 1 above)
Comment (lundibundi) [inline jstp/src/test/java/com/metarhia/jstp/connection/JSTPConnectionTest.java]: @aqrln lol, thanks for catching that.

**Comment by lundibundi** [comment]
Landed in 0671f0f60fd2f8507164b113ddbac6a73f406e79.

File: jstp/src/test/java/com/metarhia/jstp/connection/JSTPConnectionTest.java (hunk 2 above)
Comment (lundibundi) [inline jstp/src/test/java/com/metarhia/jstp/connection/JSTPConnectionTest.java]: I've fixed this one as it was using mixed argument-break style. 

But rules are meant to be followed strictly, either we have the rules or we don't, there is no such thing as half-rule. So as we have decided on google java code style we have to follow it.

**Pull request closed without merging**